src/pp_25_guessing_game_two.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+
+def input_validator(input_number: int) -> bool:
+    """
+    user guess validator
+
+    Args:
+        input_guess (int): user input integer
+
+    Returns:
+        bool: approval of the validation
+    """
+    if 1 <= input_number <= 9:
+        return True
+
+    return False
+
+
+def guess_the_number(input_number: int, guess_number: int) -> dict:
+    """
+    Args:
+        input_number (int): user input number
+        guess_number (int): by programe guess number
+
+    Returns:
+        dict: validation
+    """
+    if not input_validator(input_number):
+        print("You have to give me the number between 1 and 9 (included). Try again!")
+        return False
+
+    return evaluate_guess(input_number, guess_number)
+
+
+def evaluate_guess(user_input: int, guess_number: int) -> dict:
+    """
+    guess and input number evaluation
+
+    Args:
+        user_input (int): user input number
+        guess_number (int): by programme guessed number
+
+    Returns:
+        dict: dictionary with min and max value
+    """
+    min_num = 1
+    max_num = 9
+
+    if user_input == guess_number:
+        print("The number is {number}. You won!".format(number=user_input))
+        return {
+            'min' : min_num,
+            'max' : max_num
+            }
+
+    if guess_number < user_input:
+        min_num = guess_number + 1
+        print("Your guess {guess} is lower than the number. Guess again".format(guess=guess_number))
+    elif guess_number > user_input:
+        max_num = guess_number - 1
+        print("Your guess {guess} is higher than the number. Guess again".format(
+            guess=guess_number))
+
+    return {
+        'min' : min_num,
+        'max' : max_num
+        }
+
+
+def guess_game_two():
+    """
+    starter of the guess game two
+    """
+    user_input = int(input("Give me the number between 1 and 9 (included): "))
```

Comment:
Please call user input validation directly under this input. - In case the input is wrong - there is no point to continue.